RecoPixelVertexing/PixelTrackFitting/src/PixelFitterByHelixProjections.cc

```diff
@@ -104,36 +104,20 @@ reco::Track* PixelFitterByHelixProjections::run(
   int nhits = hits.size();
   if (nhits <2) return 0;
 
-  vector<GlobalPoint> points(nhits);
-  vector<GlobalError> errors(nhits);
-  vector<bool> isBarrel(nhits);
-  
-  if (theTrackerWatcher.check(es)) {
-    edm::ESHandle<TrackerGeometry> trackerESH;
-    es.get<TrackerDigiGeometryRecord>().get(trackerESH);
-    theTracker = trackerESH.product();
-  }
-
-  if (theFieldWatcher.check(es)) {
+  {  
```

Comment:
This change is fine with me. However, the ESWatcher theFieldWatcher was guaranteeing that theField was being updated on IOV changes.